exporters/fluentd/include/opentelemetry/exporters/fluentd/common/fluentd_common.h

```diff
@@ -99,6 +100,93 @@ void inline PopulateAttribute(
   }
 }
 
+void inline PopulateOwnedAttribute(
```

Comment:
nit: wrap this function `PopulateOwnedAttribute` in the new experimental macro, or we'll get an unused function for build without enabling the experimental flag?